### PR TITLE
Add group incentives.

### PR DESCRIPTION
**Pull request by eschlange**
* Updated home view.
* Updated styles to work with view changes.
* Added images to incentive lists.
* Added model, controller and helper methods.
* Added applicable specs.
* Added community plot positions.
* Updated admin page for incentives.

[#92252156]

### CODE REVIEW

File: app/helpers/social_networking/incentives_helper.rb [hunk 1]
```diff
@@ -11,5 +11,10 @@ def incentive_behavior_complete_count(participant_incentive)
       total_behaviors = participant_incentive.incentive.behaviors.count
       "#{complete_behaviors}/#{total_behaviors} complete"
     end
+
+    def admin_incentive_description(incentive)
+      "#{image_tag incentive.image_url, size: '50x50'}" \
```
Comment (michaelwehrley) [inline app/helpers/social_networking/incentives_helper.rb]: Does it makes sense to use `PROFILE_PAGE_LIST_DIMENSIONS` here?

File: app/models/content_providers/feed_tool_provider.rb [hunk 2]
```diff
@@ -9,11 +9,33 @@ def render_current(options)
         locals: feed_data_for(options.participant,
                               options.view_context.social_networking)
                   .merge(garden_participants:
-                           active_profiles(options.participant))
-                  .merge(individual_garden_image_dimensions: "50x50")
+                           active_profiles(options.participant),
+                         individual_garden_image_dimensions: "50x50",
```
Comment (michaelwehrley) [inline app/models/content_providers/feed_tool_provider.rb]: and here... `PROFILE_PAGE_LIST_DIMENSIONS` etc?

File: app/models/sunnyside/incentive.rb [hunk 3]
```diff
@@ -33,5 +59,27 @@ def create_group_incentives
         )
       end
     end
+
+    def complete?
+      complete = true
```
Comment (michaelwehrley) [inline app/models/sunnyside/incentive.rb]: I think this achieves the same thing: `active_participant_incentives.map(&:is_complete).all?`

**Comment by michaelwehrley** [comment]
:eyes: good.  I just made a few comments.

**Comment by eschlange** [comment]
Thanks for the comments, Mike! Would you please take a look at my changes, with regards to your comments, and merge if everything looks good?

**Comment by michaelwehrley** [comment]
Yay!  looks good!

**Pull request merged**